packages/trackerless-network/src/logic/NodeList.ts

```diff
@@ -6,6 +6,8 @@ import { EventEmitter } from 'eventemitter3'
 export interface Events {
     nodeAdded: (id: DhtAddress, remote: ContentDeliveryRpcRemote) => void
     nodeRemoved: (id: DhtAddress, remote: ContentDeliveryRpcRemote) => void
+    // !! notice: the params are not used for anything, this is a workaroud for a typing bug !!
```

Comment:
find a better way of doing this

Reply:
Yes the bug should be fixed it's very annoying